crates/core/permissions/src/models/server.rs

```diff
@@ -1,5 +1,8 @@
+use bson::Bson;
+use revolt_rocket_okapi::{revolt_okapi::schemars, JsonSchema};
```

Comment:
Use
```rust
#[cfg_attr(feature = "schemars", derive(JsonSchema))]
```

However, it is necessary to add these 3 lines to lib.rs: https://github.com/revoltchat/backend/blob/master/crates/core/models/src/lib.rs#L5